lib/src/screens/settings/security_backup_page.dart

```diff
@@ -27,44 +27,32 @@ class SecurityBackupPage extends BasePage {
       child: Column(mainAxisSize: MainAxisSize.min, children: [
         SettingsCellWithArrow(
           title: S.current.show_keys,
-          handler: (_) => Navigator.of(context).pushNamed(Routes.auth,
-              arguments: (bool isAuthenticatedSuccessfully, AuthPageState auth) {
-            if (isAuthenticatedSuccessfully) {
-              auth.close(route: Routes.showKeys);
-            }
-          }),
+          handler: (_) => context.navigateToAuthenticatedRoute(route: Routes.showKeys),
```

Comment:
and then here you can inject the AuthService dependency and call it here
`_authService.authenticateAction(route: Routes.showKeys),